mne/evoked.py

```diff
@@ -476,8 +480,13 @@ def plot_topomap(self, times="auto", ch_type=None, layout=None, vmin=None,
             but vmax is not, defaults to np.max(data).
             If callable, the output equals vmax(data).
         cmap : matplotlib colormap | None
-            Colormap to use. If None, 'Reds' is used for all positive data,
-            otherwise defaults to 'RdBu_r'.
+            Colormap to use. If 'interactive', the colors are adjustable by
+            clicking and dragging the colorbar with left and right mouse
+            button. Left mouse button moves the scale up and down and right
+            mouse button adjusts the range. Up and down arrows can be used to
+            change the colormap. Interactive mode works smoothly only for a
+            small amount of topomaps. If None (default), 'Reds' is used for all
+            positive data, otherwise defaults to 'RdBu_r'.
```

Comment:
That makes sense for my second question about always interactive. But we still have a problem related to my first question about both setting the cmap and making it interactive, right?